setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = tiledb-ml
-version = 0.2.2
+version = 0.2.3
```

Comment:
Minor question unrelated to this PR directly, can we use something like setuptools_scm to set the version automatically? We use it other projects and it works quite well.

Reply:
Yeap!